components/MatterInterface/MatterInterface.cpp

```diff
@@ -104,19 +120,48 @@ static esp_err_t app_identification_cb(identification::callback_type_t type, uin
     return ESP_OK;
 }
 
-static esp_err_t app_attribute_update_cb(callback_type_t type, uint16_t endpoint_id, uint32_t cluster_id,
-                                         uint32_t attribute_id, esp_matter_attr_val_t *val, void *priv_data)
+/**
+ * @brief This API should be called to update the driver for the attribute being updated.
+ * This is usually called from the common `app_attribute_update_cb()`.
+ * @param[in] endpoint_id Endpoint ID of the attribute.
+ * @param[in] cluster_id Cluster ID of the attribute.
+ * @param[in] attribute_id Attribute ID of the attribute.
+ * @param[in] val Pointer to `esp_matter_attr_val_t`. Use appropriate elements as per the value type.
+ * @return ESP_OK on success.
+ * @return error in case of failure.
+ */
+static esp_err_t app_attribute_update_cb(callback_type_t type, 
+    uint16_t endpoint_id, uint32_t cluster_id,                                         
+    uint32_t attribute_id, esp_matter_attr_val_t *val, void *priv_data)
 {
-    if (type == PRE_UPDATE)
+     esp_err_t err = ESP_OK;
+
+    if (type == PRE_UPDATE)//update before save in database(usually drivers)
     {
-        /* Handle the attribute updates here. */
+        app_driver_handle_t driver_handle = (app_driver_handle_t)priv_data;
+        
+#ifdef CONFIG_DONE_MATTER_DEVICE_COFFEE_MAKER            
+        err = DoneCoffeeMakerAttributeUpdate(
+                driver_handle, endpoint_id, 
+                cluster_id, attribute_id, val);
+#endif                
     }
+    if (type == POST_UPDATE) 
+    {
+        //TODO: update after save in database
+    }    
 
     InterfaceHandler->MatterAttributeUpdateCB(type, endpoint_id, cluster_id, attribute_id, val, priv_data);
-
-    return ESP_OK;
+    
+    return err;  
 }
 
+/** 
+ * @brief this API call from app_main to create main task of Matter
+ * @param[in] MatterInterfaceHandler Matter task handler
+ * @return ESP_OK on success.
+ * @return error in case of failure.
+ */
 bool Matter_TaskInit(MatterInterfaceHandler_t *MatterInterfaceHandler)
 {
     InterfaceHandler = MatterInterfaceHandler;
```

Comment:
In this here you config Matter service and you don't creat a specific task for Matter, please change his name

Reply:
all of the Matter tasks(thread) were created internally. this task is started point of them with this API (esp_matter::start(app_event_cb);)